src/pymatgen/io/aims/inputs.py

```diff
@@ -763,55 +836,62 @@ def __init__(
                 a name of a basis set (`light`, `tight`...) or a mapping from site labels to basis set names.
                 The name of a basis set can either correspond to the subfolder in `defaults_2020` folder
                 or be a full path from the `FHI-aims/species_defaults` directory.
+            species_dir (str | Path | None): The base species directory
             elements (dict[str, str] | None):
                 a mapping from site labels to elements. If some label is not in this mapping,
                 it coincides with an element.
         """
         super().__init__()
         self.labels = labels
         self.basis_set = basis_set
+        self.species_dir = species_dir
         if elements is None:
             elements = {}
         self.elements = {}
         for label in self.labels:
+            if ",spin" in label:
+                label = label.split(",")[0]
```

Comment:
seems brittle to rely on the comma in `",spin"` being the first comma in `label`

Reply:
True I replaced it with an re.sub call